### ISSUE TITLE
support 'skip' as test result

### ISSUE THREAD

**Issue by mwasilew** [Linaro/squad#28]
'skip' is not recognized as a test result value. Apart from that I think we should rather ignore unrecognized values than crash.

```
May 26 10:25:49 qa-reports squad-admin[15661]: [2017-05-26 10:25:49 +0000] [ERROR] Task squad.ci.tasks.fetch[7aa1d5e0-d249-4669-b3a2-71cc493186c1] raised unexpected: InvalidTestsData('\'skip\' is not a valid test result. Only "pass" and "fail" are accepted',)
May 26 10:25:49 qa-reports squad-admin[15661]: Traceback (most recent call last):
May 26 10:25:49 qa-reports squad-admin[15661]:   File "/srv/qa-reports.linaro.org/lib/python3.5/site-packages/celery/app/trace.py", line 240, in trace_task
May 26 10:25:49 qa-reports squad-admin[15661]:     R = retval = fun(*args, **kwargs)
May 26 10:25:49 qa-reports squad-admin[15661]:   File "/srv/qa-reports.linaro.org/lib/python3.5/site-packages/celery/app/trace.py", line 438, in __protected_call__
May 26 10:25:49 qa-reports squad-admin[15661]:     return self.run(*args, **kwargs)
May 26 10:25:49 qa-reports squad-admin[15661]:   File "/srv/qa-reports.linaro.org/lib/python3.5/site-packages/squad/ci/tasks.py", line 23, in fetch
May 26 10:25:49 qa-reports squad-admin[15661]:     test_job.backend.fetch(test_job)
May 26 10:25:49 qa-reports squad-admin[15661]:   File "/srv/qa-reports.linaro.org/lib/python3.5/site-packages/squad/ci/models.py", line 48, in fetch
May 26 10:25:49 qa-reports squad-admin[15661]:     self.really_fetch(test_job)
May 26 10:25:49 qa-reports squad-admin[15661]:   File "/srv/qa-reports.linaro.org/lib/python3.5/site-packages/squad/ci/models.py", line 69, in really_fetch
May 26 10:25:49 qa-reports squad-admin[15661]:     metrics_file=json.dumps(metrics),
May 26 10:25:49 qa-reports squad-admin[15661]:   File "/srv/qa-reports.linaro.org/lib/python3.5/site-packages/squad/core/tasks/__init__.py", line 87, in __call__
May 26 10:25:49 qa-reports squad-admin[15661]:     validate(metadata_file, metrics_file, tests_file)
May 26 10:25:49 qa-reports squad-admin[15661]:   File "/srv/qa-reports.linaro.org/lib/python3.5/site-packages/squad/core/tasks/__init__.py", line 32, in __call__
May 26 10:25:49 qa-reports squad-admin[15661]:     self.__validate_tests__(tests_file)
May 26 10:25:49 qa-reports squad-admin[15661]:   File "/srv/qa-reports.linaro.org/lib/python3.5/site-packages/squad/core/tasks/__init__.py", line 74, in __validate_tests__
May 26 10:25:49 qa-reports squad-admin[15661]:     raise exceptions.InvalidTestsData.value(value)
May 26 10:25:49 qa-reports squad-admin[15661]: squad.core.tasks.exceptions.InvalidTestsData: 'skip' is not a valid test result. Only "pass" and "fail" are accepted

```

**Comment by mwasilew** [comment]
Test object has a boolean field which only allows to store 'pass' or 'fail'.

    result = models.BooleanField()

**Comment by terceiro** [comment]
this was fixed by 49a090e
